app/controllers/api/v1/classifications_controller.rb

```diff
@@ -34,6 +34,14 @@ def project
     index
   end
 
+  def offset
+    @controlled_resources = Classification.where("project_id = ? AND id > ?", params[:project_id], params[:last_id])
+    .includes(:project)
+    .includes(:user)
+    render json_api: serializer.page(params, controlled_resources, context),
```

Comment:
specifically the project scope for rules in this [line](https://github.com/zooniverse/Panoptes/blob/50566215d7d263df52a307a2b433d2185623153f/app/models/classification.rb#L39)